mkdocs.yml

```diff
@@ -288,6 +288,7 @@ nav:
       - Libraries: reference/libraries/README.md
       - Glossary: reference/glossary/README.md
   - Releases Notes:
+      - 2.1.7: releases/2.1.7.md
```

Comment:
Fixes this: https://github.com/OrchardCMS/OrchardCore/actions/runs/14538214228/job/40790823850